src/js/events/AppsEvents.js

```diff
@@ -14,7 +14,8 @@ const AppsEvents = {
   SCALE_APP_ERROR: "APPS_EVENTS_SCALE_APP_ERROR",
   APPLY_APP: "APPS_EVENTS_APPLY_APP",
   APPLY_APP_ERROR: "APPS_EVENTS_APPLY_APP_ERROR",
-  APPLY_APP_REQUEST: "APPS_EVENTS_APPLY_APP_REQUEST"
+  APPLY_APP_REQUEST: "APPS_EVENTS_APPLY_APP_REQUEST",
+  APPS_STATUSES: "APPS_STATUSES"
```

Comment:
As we are in AppsEvents i think a better name might  be: `STATUS_UPDATE` ... what do you think?